kairon/api/app/routers/bot/bot.py

```
@@ -840,6 +840,18 @@ async def get_action_server_logs(start_idx: int = 0, page_size: int = 10,
     }
     return Response(data=data)
 
+@router.get("/parallel/{name}/logs", response_model=Response)
+async def get_parallel_action_logs(
+    name: str,
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff (0.11.9)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)